.github/CODE_OF_CONDUCT.md

```diff
@@ -1,6 +1,6 @@
 # Community Code of Conduct
 
-Please see the official [Ansible Community Code of Conduct][CoC].
+Please see the official [Ansible Community Code of Conduct][coc].
 
-[CoC]:
+[coc]:
```

Comment:
I asked you not to do the lowercasing in another PR :man_facepalming: Why did you sneak it in anyway?